web/src/validators/ChangePasswordValidator.ts

```diff
@@ -0,0 +1,21 @@
+import { Validator } from "fluentvalidation-ts";
+import { ChangePasswordFormData } from "../pages/ProfilePage/components/ChangePassword/ChangePassword";
+
+export class ChangePasswordValidator extends Validator<ChangePasswordFormData> {
+    passwordPattern = new RegExp(/^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(\S).{8,20}$/)
+
+    constructor() {
+        super();
+
+        this.ruleFor("oldPassword")
+            .notEmpty().withMessage("Password is required")
+            .matches(this.passwordPattern).withMessage("Password must contain at least 8 characters, 1 capital letter and 1 number")
+
+        this.ruleFor("newPassword")
+            .notEmpty().withMessage("Password is required")
```

Comment:
missing unique message for old and new password

Reply:
done

Reply:
I dont see a change that fixed this. Please add a unique message for both.

Reply:
it is done now